src/core/account.zig

```diff
@@ -1,6 +1,18 @@
-const Pubkey = @import("pubkey.zig").Pubkey;
-const Epoch = @import("./time.zig").Epoch;
-const AccountInFile = @import("../accountsdb/accounts_file.zig").AccountInFile;
+const sig = @import("../lib.zig");
+const bincode = sig.bincode;
+
+const Pubkey = sig.core.Pubkey;
+const Epoch = sig.core.Epoch;
+const AccountInFile = sig.accounts_db.accounts_file.AccountInFile;
+
+const U8SliceConfig = bincode.int.U8SliceConfig;
+
+pub const AccountWithoutData = struct {
```

Comment:
I probably would have called this AccountMetadata, but we have this already and it's called AccountInfo. Actually this struct isn't even used so I would just delete it.